blog/_posts/2021-05-05-624-and-2021.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "ROOT News: v6.24, what 2021 will bring, and 404"
```

Comment:
I feel like "404" in the title is a bit confusing, might just be me. Maybe we could hint at it by phrasing it like "...what 2021 will bring, and a surprise" or "..., and a [friendly] competition"